doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,150 @@
+## Main differences between SLE12 and SLE15 profiles
+
+### Product selection
+
+- https://github.com/SUSE/doc-sle/blob/develop/xml/ay_bigfile.xml#L4521
+
+### Firewall configuration
+
+https://github.com/SUSE/doc-sle/blob/develop/xml/ay_bigfile.xml#L12999
+
+In SLE15, SuSEfirewall2 has been replaced by firewalld as the default firewall.
```

Comment:
I guess we also can point to the firewalld official configuration, as it's easy to map ay options to the firewalld.

Reply:
It was added as a link for further documentation :heavy_check_mark: